coral/src/app/features/configuration/environments/hooks/getPaginatedEnvironments.tsx

```diff
@@ -0,0 +1,66 @@
+import { useQuery } from "@tanstack/react-query";
```

Comment:
I think we should test the hook. Then we maybe don't even need to mock api calls in tests, but can mock the hook response?

Reply:
I'm not opposed to it, but what would a useful test be? I'm not sure what I want to ascertain when testing such a thin wrapper over a `useQuery`, which I am confident is itself well tested by the library authors :o

Reply:
In this case it's not really useful, since the hook isn't doing anything really, other then wrapping stuff.  It would be only useful to start a pattern, because in other cases the hook could do more inside and then the test for the hook could cover that. Also everything comes from one api, so we don't have to remember extra mocks. 

In that case (e.g. hook doing more stuff / calling differently located apis) it could be nice to have the hook tested explicitly and then in tests only to mock the hook, not the api responses. 

So I would say leave it as it is and when we have a case where we think it makes sense, we can still do it :D